modules/apps/push-notifications/push-notifications-sender/push-notifications-sender-apple/src/main/java/com/liferay/push/notifications/sender/apple/internal/util/ApplePushNotificationsMessagingConfiguratorUtil.java

```diff
@@ -3,28 +3,32 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.push.notifications.sender.apple.internal.messaging;
+package com.liferay.push.notifications.sender.apple.internal.util;
 
 import com.liferay.portal.kernel.messaging.MessageListener;
 import com.liferay.portal.kernel.util.MapUtil;
 import com.liferay.push.notifications.constants.PushNotificationsDestinationNames;
 
+import com.liferay.push.notifications.sender.apple.internal.messaging.ApplePushNotificationsResponseMessageListener;
+import org.osgi.framework.Bundle;
 import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
 import org.osgi.framework.ServiceRegistration;
-import org.osgi.service.component.annotations.Activate;
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Deactivate;
 
 /**
  * @author Bruno Farache
  */
-@Component(
-	enabled = false, service = ApplePushNotificationsMessagingConfigurator.class
-)
-public class ApplePushNotificationsMessagingConfigurator {
+public class ApplePushNotificationsMessagingConfiguratorUtil {
```

Comment:
Hey @RenanUmV, just merge it with `ApplePushNotificationsResponseMessageListener`. Take a look at others *MessageListener classes and apply the pattern.